src/i18n/en.json

```diff
@@ -59,7 +59,8 @@
       "close-lbl_warning": "OK",
       "close-lbl_success": "OK",
       "close-lbl_error": "Close",
-      "close-lbl_info": "Got it"
+      "close-lbl_info": "Got it",
+      "idle-notification-message": "You have been logged out due to long inactivity!"
```

Comment:
Use the right translations. See [below](https://github.com/tokend/web-client/pull/74/files#diff-f0cb016d0d4f26ee2a23b1539d1e7d4bR12) what exactly I mean